lib/resources/majors.tsx

```diff
@@ -2255,17 +2255,12 @@ const bsCS_Old: Major = {
         {
           description:
             '<b>Upper Level CS Credits</b> <br /> ' +
-            'At least 13 more upper level CS credits are required. ' +
-            'At least one course in two different classification areas (Applications, Reasoning, Software, Systems) must be chosen in addition to Theory (Algorithms).', // TODO: Question: How to include this requirement?: SEPARATE DISTRIBUTION PATHING
+            'At least 13 more upper level CS credits are required. ',
           required_credits: 13,
           criteria: 'EN Computer Science[D]^AND^Upper Level Undergraduate[L]',
           double_count: [
             '<b>Team Requirement</b> <br /> ' +
               'Select one course with Program of Study Tag CSCI-TEAM.',
-            '<b>CS Electives</b> <br /> ' +
-              'Eight additional credits of Computer Science are required.' +
-              'For an approved list of courses from other departments (maximum of 6 credits allowed), visit https://www.cs.jhu.edu/computer-science-other-courses-for-bs-degree/ ' +
-              'and create a custom course to satisfy this requirement.',
```

Comment:
`CS Electives` can overlap with `Upper Level CS Credits` but not vice versa?

Reply:
Should not be possible either way. Electives need to be exclusive from everything else

Reply:
ohh my bad I was looking at the wrong changes